corelib-db/src/main/java/eu/europeana/corelib/db/service/UserService.java

```diff
@@ -7,7 +7,6 @@
 import eu.europeana.corelib.definitions.db.entity.relational.SocialTag;
 import eu.europeana.corelib.definitions.db.entity.relational.Token;
 import eu.europeana.corelib.definitions.db.entity.relational.User;
-import eu.europeana.corelib.neo4j.exception.Neo4JException;
 import eu.europeana.corelib.web.exception.EmailServiceException;
 
 import java.util.List;
```

Comment:
Can't we delete this file altogether? I suppose that now the apikeyservice integration is live we not just won't use it any longer, but even can't use it any more.

Reply:
My idea was to remove everything after the Search & Record API starts using the new API key service in production.